src/double_pendulum_simulator.ipynb

```diff
@@ -34,6 +34,7 @@
     ")\n",
     "from planar_rigid_body_visualizer import PlanarRigidBodyVisualizer\n",
     "from manipulator_dynamics import manipulator_dynamics"
+    "import underactuated_utils as utils"
```

Comment:
n.b. - somehow this edit broke the notebook (it was missing the comma on the line above).  not sure how that happened, and i'm

Reply:
OK good to add unit test